SmartBook/SmartBook/View/Activity/AppointmentListViewModel.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  AppointmentListViewModel.swift
+//  SmartBook
+//
+//  Created by Md. Kamrul Hasan on 8/10/25.
+//
+
+import Foundation
+
+final class AppointmentListViewModel {
+    
+    private(set) var appointments: [Appointment] = []
+    private let displayFormatter: DateFormatter
+    
+    init(displayFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "dd MMM yyyy, h:mm a"
+        return formatter
+    }()) {
+        self.displayFormatter = displayFormatter
+    }
+    
+    func loadAppointments(completion: @escaping () -> Void) {
+        SmartBookingService.shared.fetchAppointments { [weak self] appointments, _ in
+            guard let self = self else { return }
```

Comment:
Please use modern syntax - 
```suggestion
            guard let self else { return }
```